hw4/index.html

```diff
@@ -0,0 +1,134 @@
+            <tr>
+                <td><i>Italic</i></td>
+                <td><abbr title="text">Abbreviation</abbr></td>
+                <td><s>Delete</s></td>
+            </tr>
+            <tr>
+                <td><u>Underline</u></td>
+                <td>5<sup>3</sup>=125</td>
+                <td><ins>Inserted</ins></td>
+            </tr>
+        </table>
+        <h2 id="part3">Unordered List</h2>
+        <ul>
+            <li>List item 1</li>
+            <li>List item 2</li>
+                <ul>
+                    <li>Some nested list item 1</li>
+                </ul>
+            <li>List item 3</li>
+        </ul>
+        <h2>Ordered List</h2>
+        <ol>
+            <li>List item 1</li>
+            <li>List item 2</li>
+                <ol class="alpha">
+                    <li>Some nested list item 1</li>
+                </ol>
+            <li>List item 3</li>
+        </ol>
+        <h3>Definition List</h3>
+        <dl>
+            <dt>defines terms 1</dt>
+            <dd>description 1</dd>
+            <br>
+            <dt>defines terms 2</dt>
+            <dd>description 2</dd> 
+        </dl>
+        <h2>Details</h2>
+            <details open>
+            <summary>Some details element (open)</summary>
+                Here are more details about the details.
+            </details>
+            <details>
+            <summary>Some details element (close)</summary> 
+                Here are more details about the details.
+            </details>
+        <h2 id="part4">Text formats</h2>
+        <p><b>Article Text</b></p>
+
+        <p style="font-size: 20px;"><strong>Article title</strong></p>
+
+        <p><small>Article description</small></p>
+
+        <h3>Preformatted Text</h3>
+            <pre><p class="mach">Text
+  in a pre element
+    is displayed in a fixed-width
+font, and it preserves
+both           spaces and
+    line breaks</p></pre>
+
+    <h3>Code</h3>
+        <div class="cd">
+            <p class="code">
+                <code>
+                    .wrapper { <br>
+                    display: flex; <br>
+                    vertical-align: middle; <br>
+                    justify-content: center;<br>
+                    }
+                </code>
+            </p>
+        </div>
+
+    <h2 id="part5">Blockquotes</h2>
+        <p>Let’s keep it simple. Italics are good to help set it off from the body text.
+             Be sure to style the citation.</p>
+        <blockquote><i>"If you go back a few hundred years, what we take for granted today 
+            would seem like magic-being able to talk to people over long distances, to transmit 
+            images, flying, accessing vast amounts of data like an oracle. These are all 
+            things that would have been considered magic a few hundred years ago."<br>         
+            <a href="#">Elon Musk</a></i></blockquote>
+</main>
+<footer><p class="ftr"><span class="i"></span>Copyright Epam Systems</p></footer>

```

Comment:
You can try to add copyright symbol here (without styles)
https://blog.hubspot.com/website/html-code-copyright#:~:text=To%20add%20a%20copyright%20symbol,it%20appear%20on%20a%20page.

Reply:
done